src/dataset/openea_dataset.py

```diff
@@ -29,7 +33,6 @@ def __init__(self, data_folder: str, division: str, args_path: str):
         )
         self._name = data_folder.split("/")[-2] + "/" + division[:-1]
 
-    @abstractmethod
```

Comment:
This should not have been here right? Because it is actually implemented...

Reply:
hmm, odd. No, I guess it shouldn't. Especially since OpenEaDataset is a concrete class